src/docfx/config/ops/OpsPreProcessor.cs

```diff
@@ -49,10 +49,10 @@ private void PreProcessMonoDocXml()
 
                         var fallbackXmlPath = _buildOptions.FallbackDocsetPath is null
                             ? null
-                            : Path.Combine(_buildOptions.FallbackDocsetPath.Value, monodocConfig.SourceXmlFolder);
+                            : PathUtility.Normalize(Path.Combine(_buildOptions.FallbackDocsetPath.Value, monodocConfig.SourceXmlFolder));
```

Comment:
why is this normalization needed? Does the format comply with `ECMA2YAML` plugin?

Reply:
`SourceXmlFolder` is relative to docset root, and `fallbackXmlPath` is not normalized in Ecma2Yaml before used as a trimming-prefix. (ecma2yaml use plain string to trim leading path)

e.g. 
![image](https://user-images.githubusercontent.com/42199316/84229304-d852e780-ab1b-11ea-9e89-0ded57d8580e.png)

Reply:
Would `Path.GetFullPath` suffice?